tests/configs/test_zoo.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from simsopt.configs import get_QUASR_data
+
+class ZooTests(unittest.TestCase):
```

Comment:
Just noting that, as you're hitting an external resource (QUASR website), this isn't strictly a unit test but an integration test. Now, an integration test may be what you actually care about, but there's always the risk that it will fail (or start failing) due to factors that don't indicate an error in your code.
I would consider mocking the `requests.get` call for regular testing, and have a separate integration test that's run less frequently.